src/styles/variables.scss

```diff
@@ -0,0 +1,6 @@
+$BorderColor: #f3f3f3;
+$blackColor: #060b35;
+$grayColor: #616070;
+$blueColor: #00acdc;
+$whiteColor: #fff;
+$cardWidth: 200px;
```

Comment:
It is not a good practice to name variable by their colors.
I recommend doing like this:
```
$color__buy-background: #00acdc;
$color__buy-background--hover: #fff;
```

This way you understand the purpose of the color.